pkg/mhooks/event_type.go

```diff
@@ -35,6 +39,165 @@ type AccountCreated struct {
 	AccountID string `json:"accountID"`
 }
 
+type AccountDeleted struct {
+	// ID of the account
+	AccountID string `json:"accountID"`
+}
+
+type AccountUpdated struct {
+	// ID of the account
+	AccountID string `json:"accountID"`
+}
+
+type BalanceUpdated struct {
+	// ID of the Account associated with the wallet
+	AccountID string `json:"accountID"`
+	// ID of the Wallet
+	WalletID string `json:"walletID"`
+}
+
+type BankAccountCreated struct {
+	// ID of the bank account
+	BankAccountID string `json:"bankAccountID"`
+	// ID of the account where the bank account was created
+	AccountID string `json:"accountID"`
+}
+
+type BankAccountDeleted struct {
+	// ID of the bank account
+	BankAccountID string `json:"bankAccountID"`
+	// ID of the account where the bank account was deleted
+	AccountID string `json:"accountID"`
+}
+
+type BankAccountUpdated struct {
+	// ID of the bank account
+	BankAccountID string `json:"bankAccountID"`
+	// ID of the account where the bank account was updated
+	AccountID string `json:"accountID"`
+	// Status of the bank account
+	Status           moov.BankAccountStatus       `json:"status"`
+	StatusReason     moov.BankAccountStatusReason `json:"statusReason"`
+	ExceptionDetails *moov.ExceptionDetails       `json:"exceptionDetails,omitempty"`
+}
+
+type CardAutoUpdated struct {
+	// ID of the merchant's Account associated with the updated Card
+	AccountID string `json:"accountID"`
+	// ID of the Card that was updated
+	CardID     string              `json:"cardID"`
+	UpdateType moov.CardUpdateType `json:"updateType"`
+}
+
+type CapabilityRequested struct {
+	Capability moov.CapabilityName `json:"capabilityID"`
```

Comment:
JSON name here should just be `capability` and have the `ID` suffix omitted.. but that needs to be patched in our internal API / webhooks first.